README.md

```diff
@@ -51,7 +65,7 @@ SIM is an open source platform for carrying out algorithmic contests
   ```sh
   make install
   ```
-  It will ask for MySQL credentials
+  It will ask for MySQL credentials. For testing purposes, you can type sim, sim, sim, localhost for MariaDB username, password, database and user host.
```

Comment:
Maybe correlate to the step 3. (creating MySQL database and user) if it was taken?

Reply:
Maybe a better option is to make this step automatic; make those options default (pressing enter applies default options) in make install?

Reply:
Maybe, but adding a note like "Hey, these are credentials to the MySQL account you have just created in point 3 (if you created the account that way)." could be helpful as well. What do you think?